src/v/cluster/BUILD

```diff
@@ -251,6 +251,18 @@ redpanda_cc_library(
     visibility = ["//visibility:public"],
 )
 
+redpanda_cc_library(
```

Comment:
This is also available as separate PR https://github.com/redpanda-data/redpanda/pull/23889